### ISSUE TITLE
[feature request] Tab mode, and more

### ISSUE THREAD

**Issue by NewBugger** [zhanghai/MaterialFiles#238]
1. add a Tab View mode, not only the current New Window mode.

2. Common Computer Code (like json, sh script, html, etc) syntax highlighted color. and enhanced Text Editor (like Mixplorer file manager)

3. UI modification: current view vision (file list view) is too big at small-screen phone, please consider zoom out the view scale.

4. Do Not save history visited directory at the current access dir bar.

5. restore the last access at new start (like Windows file manager)

I'm very loving in the `prefer normal access` feature, so please make this app better, thanks.

**Comment by magnus-gross** [comment]
1. is a duplicate of #181

**Comment by NewBugger** [comment]
@magnus-gross  ok, and hopes the others could be applied.

**Comment by MuntashirAkon** [comment]
I disagree with number 4. This is a nice feature and all popular FM has this feature.

**Comment by NewBugger** [comment]
> I disagree with number 4. This is a nice feature and all popular FM has this feature.

if i'm on a directory B now, i'll no need to keep a directory A history on my access dir bar, it makes user disorted. it should be placed in a separate history menu. and i've never seen this design in other explorer like FS and Mixplorer.

**Comment by MuntashirAkon** [comment]
> it makes user disorted.

No, it doesn't. Mixplorer has some nice features but it's UI looks crappy.
